Formula/mpfrcx.rb

```diff
@@ -0,0 +1,36 @@
+class Mpfrcx < Formula
+  desc "Arbitrary precision library for arithmetic of univariate polynomials"
+  homepage "https://www.multiprecision.org/mpfrcx/home.html"
+  url "https://www.multiprecision.org/downloads/mpfrcx-0.6.3.tar.gz"
+  sha256 "9da9b3614c0a3e00e6ed2b82fc935d1c838d97074dc59cb388f8fafbe3db8594"
+  license "GPL-3.0-or-later"
+  head "https://gitlab.inria.fr/enge/mpfrcx.git", branch: "master"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+  depends_on "gmp"
+  depends_on "libmpc"
+  depends_on "mpfr"
+
+  def install
+    # Regenerate configure to avoid building libraries with flat namespaces
+    system "autoreconf", "--force", "--install", "--verbose"
```

Comment:
I think `-fiv` is the one that we used across the board.
```suggestion
    system "autoreconf", "-fiv"
```

Reply:
I think it differs a bit across the board? Personally, I find the long options to be more readable (and it's not like users will be going through the hassle of typing these options out on their own).

Other formulae using the long options include `afflib`, `apophenia`, `augeas`, `axel`, `bashdb`, `btfs`, `byobu`, `cddlib`, `classads`, `cmockery`, `cpmtools`, `ctail`, `cvs`, `ddrescue`, `duti`, `dynare`, `ekhtml`, `epsilon`, `expect`, and a lot more.

I'm not too opinionated on this, but just wanted to point out that there are existing formulae already using this format. I recall @carlocab did have some thought/opinion on this in the past, if there's a desire to standardize on this (long vs short options). Otherwise I'm fine with just having a mix of short/long `autoreconf` options in `homebrew-core`.

Reply:
yeah, that is fine, just saying the style differences.

Reply:
Long options are better for readability and (human-)parse-ability.